src/helpers/validation.ts

```diff
@@ -0,0 +1,75 @@
+import {isEmpty as isEmptyLodash} from 'lodash';
+
+const OTP_LENGTH = 6;
+const USERNAME_ATLEAST = 6;
+const PASSWORD_ATLEAST = 8;
+
+/**
+ * check if the input is empty or not
+ * @param text is input of string
+ */
+function isEmpty(text: string) : boolean {
+    if (!text) {
+      return true;
+    }
+  
+    if (typeof text === 'string') {
+      if (text.trim()) {
+        return false;
+      }
+      return true;
+    } else {
+      return isEmptyLodash(text);
+    }
+  };
+
+  /**
+   * Check user name contain only letters or digits and has atleast 6 charactors
+   * @param username is input string
+   */
+export function checkUserName(username: string) : boolean {
+    if (isEmpty(username)) {
+      return false
+    }
+    
+    const re = RegExp(`^[A-Za-z0-9]{${USERNAME_ATLEAST},}$`);
```

Comment:
```suggestion
    const re = RegExp(`^[A-Za-z0-9]{${userNameMaxLength},}$`);
```
should make userNameMaxLength as a parameter instead of using directly USERNAME_ATLEAST